src/styles/global.css

```diff
@@ -250,9 +250,47 @@ a.active {
   text-shadow: var(--txt-shadow);
 }
 
-.temp{
-  margin-top:20vh;
+.container .row:first-child {
+  margin-top: 20vh;
+}
```

Comment:
I don't think adding this as a global styled is a good idea. Because these are default bootstrap classes and these will be used elsewhere. It would be a good idea to move inside the component making it scoped.

Reply:
Oh yes, thanks! I didn't even notice them! Cleaning them out!